source/extensions/Sekai.SDL.Redist/Sekai.SDL.Redist.csproj

```diff
@@ -0,0 +1,69 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="Nuget">
+    <IsPackable>true</IsPackable>
+    <PackageId>Sekai.SDL.Redist</PackageId>
+    <Version>2.0.22</Version>
+    <Authors>slouken;Vignette</Authors>
+    <PackageTags>SDL;Sekai;</PackageTags>
+    <Title>Sekai.SDL.Redist</Title>
+    <Description>SDL Redistributables for Sekai based on latest release (this does not contain bindings, in which case, you should be looking for Sekai!)</Description>
+    <Copyright>2022 Vignette and the SDL Authors</Copyright>
+    <RepositoryUrl>https://github.com/libsdl-org/SDL</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/vignetteapp/Sekai</PackageProjectUrl>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Since this is just the runtime, I think we can change the target framework to `netstandard2.1`